package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "shelljs",
-  "version": "0.8.3",
+  "version": "0.8.6",
```

Comment:
Please undo this part. I will take care of bumping the version when I'm ready to cut a release.